A bunch of tests failed on osx-64 only:
```
=========================== short test summary info ============================
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_pytz_timezones_0_2_hours_ago
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_pytz_timezones_1_3_hours_ago
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_pytz_timezones_2_5_hours_ago
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_pytz_timezones_3_Today_at_9_pm
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_timezone_conversion_0_Today_at_4_25_pm
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_timezone_conversion_1_Yesterday_at_4_25_pm
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_timezone_conversion_2_Yesterday
tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_timezone_conversion_3_Today

FAILED tests/test_freshness_date_parser.py::TestFreshnessDateDataParser::test_freshness_date_with_to_timezone_setting

FAILED tests/test_timezone_parser.py::TestTimeZoneConversion::test_timezone_conversion_0_2015_12_31_10_04_AM

FAILED tests/test_timezone_parser.py::TestTimeZoneConversion::test_timezone_conversion_1_2015_12_30_10_04_AM

FAILED tests/test_utils.py::TestUtils::test_apply_timezone_from_settings_function_0

FAILED tests/test_utils.py::TestUtils::test_apply_timezone_function_0 - FileN...

FAILED tests/test_utils.py::TestUtils::test_localize_timezone_function_1 - Fi...

================= 14 failed, 23523 passed, 4 skipped in 54.71s =================
```

They all relate to time zones. 

It seems the core issue is `ModuleNotFoundError: No module named 'tzdata'` (only for Python 3.9???)

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -10,7 +10,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [py<38]
+  skip: True  # [py<39]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   entry_points:
     - dateparser-download = dateparser_cli.cli:entrance
@@ -37,7 +37,6 @@ requirements:
 # Skipping tests that require optional extras (calendars, fasttext, langdetect),
 # which are not installed by default in the minimal build.
 {% set ignore_tests = "not (" %}
-{% set ignore_tests = ignore_tests + "test_calendars or " %}
 {% set ignore_tests = ignore_tests + "test_fasttext or " %}
 {% set ignore_tests = ignore_tests + "test_langdetect" %}
 {% if win %}
@@ -52,6 +51,7 @@ test:
     - pip
     - pytest
     - parameterized
+    - convertdate >=2.2.1
   commands:
     - pip check
     - dateparser-download --help

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -36,13 +36,25 @@ requirements:
 
 # Skipping tests that require optional extras (calendars, fasttext, langdetect),
 # which are not installed by default in the minimal build.
-{% set ignore_tests = "not (" %}
-{% set ignore_tests = ignore_tests + "test_fasttext or " %}
-{% set ignore_tests = ignore_tests + "test_langdetect" %}
-{% if win %}
-{% set ignore_tests = ignore_tests + " or test_date_parser" %}
-{% endif %}
-{% set ignore_tests = ignore_tests + ")" %}
+# {% set ignore_tests = "not (" %}
+# {% set ignore_tests = ignore_tests + "test_fasttext or " %}
+# {% set ignore_tests = ignore_tests + "test_langdetect" %}
+# {% if win %}
+# {% set ignore_tests = ignore_tests + " or test_date_parser" %} 
+# {% endif %}
+# {% set ignore_tests = ignore_tests + ")" %}
+
+
+# {% set ignore_tests = "not (" %}
+# {% set ignore_tests = ignore_tests + "test_fasttext or " %}
+# {% set ignore_tests = ignore_tests + "test_langdetect" %}
+ {% set ignore_tests = " test_date_parser" %} # [win]
+# {% set ignore_tests = ignore_tests + ")" %}
+
+# {% set ignore_tests = "" %}
+# {% set ignore_tests = " --ignore=tests/async_torch_test.py" %}
+# {% set ignore_tests = ignore_tests + " --ignore=tests/async_writes_test.py" %}
+
 
 test:
   source_files:
@@ -56,13 +68,31 @@ test:
     - pip check
     - dateparser-download --help
     # Disable import test_date.py on Windows. This test use time.tzset() POSIX call 
-    - pytest tests -v --ignore=tests/test_date.py -k "{{ ignore_tests }}"
+    # - pytest tests -v --ignore=tests/test_date.py -k "{{ ignore_tests }}"
+    - pytest tests -v  # [not win]
+    - pytest tests -v --ignore=tests/test_date.py -k "{{ ignore_tests }}"  # [win]
  
   imports:
     - dateparser
     - dateparser.calendars
+    - dateparser.conf
+    - dateparser.custom_language_detection.language_mapping
+    - dateparser.data
+    - dateparser.data.languages_info
+    - dateparser.date
+    - dateparser.date_parser
+    - dateparser.freshness_date_parser
     - dateparser.languages
+    - dateparser.languages.loader
+    - dateparser.languages.validation
+    - dateparser.parser
+    - dateparser.search
+    - dateparser.search.detection
+    - dateparser.search.search
+    - dateparser.search.text_detection
+    - dateparser.timezone_parser
     - dateparser.utils
+    - dateparser.utils.strptime
 
 about:
   home: https://github.com/scrapinghub/dateparser

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,16 +1,17 @@
+{% set name = "dateparser" %}
 {% set version = "1.2.2" %}
 
 package:
-  name: dateparser
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/d/dateparser/dateparser-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 986316f17cb8cdc23ea8ce563027c5ef12fc725b6fb1d137c14ca08777c5ecf7
 
 build:
   number: 0
-  skip: True  # [py<39]
+  skip: True  # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
   entry_points:
     - dateparser-download = dateparser_cli.cli:entrance
@@ -36,25 +37,9 @@ requirements:
 
 # Skipping tests that require optional extras (calendars, fasttext, langdetect),
 # which are not installed by default in the minimal build.
-# {% set ignore_tests = "not (" %}
-# {% set ignore_tests = ignore_tests + "test_fasttext or " %}
-# {% set ignore_tests = ignore_tests + "test_langdetect" %}
-# {% if win %}
-# {% set ignore_tests = ignore_tests + " or test_date_parser" %} 
-# {% endif %}
-# {% set ignore_tests = ignore_tests + ")" %}
-
-
-# {% set ignore_tests = "not (" %}
-# {% set ignore_tests = ignore_tests + "test_fasttext or " %}
-# {% set ignore_tests = ignore_tests + "test_langdetect" %}
- {% set ignore_tests = " test_date_parser" %} # [win]
-# {% set ignore_tests = ignore_tests + ")" %}
-
-# {% set ignore_tests = "" %}
-# {% set ignore_tests = " --ignore=tests/async_torch_test.py" %}
-# {% set ignore_tests = ignore_tests + " --ignore=tests/async_writes_test.py" %}
-
+{% set deselect_tests = " --deselect=tests/test_date_parser.py::TestDateParser::test_parse_negative_timestamp_0__1484823450" %}
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_date_parser.py::TestDateParser::test_parse_negative_timestamp_1__1436745600000" %}
+{% set deselect_tests = deselect_tests + " --deselect=tests/test_date_parser.py::TestDateParser::test_parse_negative_timestamp_2__1015673450000001" %}
 
 test:
   source_files:
@@ -67,11 +52,10 @@ test:
   commands:
     - pip check
     - dateparser-download --help
-    # Disable import test_date.py on Windows. This test use time.tzset() POSIX call 
-    # - pytest tests -v --ignore=tests/test_date.py -k "{{ ignore_tests }}"
     - pytest tests -v  # [not win]
-    - pytest tests -v --ignore=tests/test_date.py -k "{{ ignore_tests }}"  # [win]
- 
+    # Disable import test_date.py on Windows. This test use time.tzset() POSIX call 
+    - pytest tests -v --ignore=tests/test_date.py {{ deselect_tests }}  # [win]
+
   imports:
     - dateparser
     - dateparser.calendars

